File: cdn-system/api/models/order.go
package models

import "time"

type Order struct {
	ID            int64     `json:"id" gorm:"primaryKey"`
	UserID        int64     `json:"user_id" gorm:"column:uid;index"`
	Type          string    `json:"type"`
	Description   string    `json:"des" gorm:"column:des"`
	Data          string    `json:"data"`
	CreatedAt     time.Time `json:"create_at" gorm:"column:create_at"`
	PaidAt        time.Time `json:"pay_at" gorm:"column:pay_at"`
	Amount        int64     `json:"amount"`
	PayType       string    `json:"pay_type" gorm:"column:pay_type"`
	MerchantOrder string    `json:"mch_order_no" gorm:"column:mch_order_no"`
	TransactionID string    `json:"transaction_id" gorm:"column:transaction_id"`
	State         string    `json:"state"`
}

func (Order) TableName() string {
	return "order"
}

File: cdn-system/cnn.net/src/Cnn.Common/Domain/Entities/Order.cs
using SqlSugar;

namespace Cnn.Domain.Entities;

[SugarTable("order")]
public class Order
{
    [SugarColumn(IsPrimaryKey = true, IsIdentity = true)]
    public int Id { get; set; }

    public int? Uid { get; set; }

    public string? Type { get; set; }

    public string? Des { get; set; }

    public string? Data { get; set; }

    public DateTime? CreateAt { get; set; }

    public DateTime? PayAt { get; set; }

    public long? Amount { get; set; }

    public string? PayType { get; set; }

    public string? MchOrderNo { get; set; }

    public string? TransactionId { get; set; }

    public string? State { get; set; }
}